scripts/update_example_schemas.py

```diff
@@ -0,0 +1,55 @@
+import json
+from pathlib import Path
+from typing import Any, Dict, Union
+
+import pyarrow.parquet as pq
+
+
+def find_root_dir() -> Path:
+    """Find root geoparquet directory
+
+    This is a more robust approach than assuming __file__ refers to a specific place
+    """
+    folder = Path(__file__).resolve()
+
+    while folder.name != "geoparquet":
```

Comment:
Note this fails if the user checked out the repo with a name other than geoparquet. Probably not all that likely to happen, but I happened to have it named differently so it didn't find it. Was easy to fix.